accord-core/src/main/java/accord/coordinate/Recover.java

```diff
@@ -172,7 +176,8 @@ public static Recover recover(Node node, Ballot ballot, TxnId txnId, Txn txn, Fu
 
     void start(Set<Id> nodes)
     {
-        node.send(nodes, to -> new BeginRecovery(to, tracker.topologies(), txnId, txn, route, ballot), this);
+        node.send(nodes, to -> new BeginRecovery(to, tracker.topologies(), txnId, txn, route, ballot),
```

Comment:
does this need to wrap? doesn't provide much greater clarity

Reply:
"Try to keep lines under 120 characters, but use good judgment. It is better to exceed this limit, than to split a line that has no natural splitting points, particularly when the remainder of the line is boilerplate or easily inferred by the reader."

Reply:
thanks, missed that